compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -1013,24 +1013,40 @@ impl TerminatorKind<'_> {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum SwitchTargetValue {
```

Comment:
That type is not used in this file, so it'd probably better live next to the `SwitchSources` type alias.

Reply:
Ah, you are also using it somewhere else... but it is not part of the MIR syntax (i.e., it is not stored inside a MIR body), so this file here doesn't seem like the right place.

Reply:
I'll move it to `basic_blocks.rs`.